### PR TITLE
Properly handle units on constraints.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6270/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6271/
Test PASSed.

**Comment by kadams54** [comment]
## QA

There's a bug where core and disk sizes aren't respected; that will need to be addressed in a separate PR. Consequently we're going to focus on the RAM constraint. In machine view, you'll need to add multiple machines with the following sizes for RAM:

1. 1024M
2. 2G
3. 3t
4. 4P

Once each machine is added, you should see the following values displayed:

1. 1.00GB
2. 2.00GB
3. 3072.00GB
4. 4194304.00GB

File: jujugui/static/gui/src/app/components/deployment-flow/machines/machines.js [hunk 1]
```diff
@@ -45,17 +45,22 @@ YUI.add('deployment-machines', function() {
...
+        let disk = constraints['root-disk'];
         let mem = constraints.mem;
         const cores = constraints.cores;
         const parts = [];
         if (cores && cpu && disk && mem) {
+          // Until we have a better way of handling the various units, just
+          // display the constraint without manipulation.
           cpu = cpu / 100;
+          /*
           disk = disk / 1024;
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/machines/machines.js]: This can be removed

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6272/
Test PASSed.

**Comment by kadams54** [comment]
:shipit: - thanks!

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6273/
Test PASSed.

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/2016

**Comment by hatched** [comment]
spurious

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**